bandit/__init__.py

```diff
@@ -2,7 +2,7 @@
 # Copyright 2014 Hewlett-Packard Development Company, L.P.
 #
 # SPDX-License-Identifier: Apache-2.0
-import pbr.version
+from importlib import metadata
```

Comment:
The import name of the backport library is `importlib_metadata`, so this should be:

```suggestion
try:
    from importlib import metadata
except ImportError:
    import importlib_metadata as metadata
```

Reply:
Also the backport has to be used on certain versions of python since the API in those versions was abruptly deprecated. And the backport provided no support for that API so it will have version conflicts with other tools